src/view/pages/Farmer/tabs/Buy/index.tsx

```diff
@@ -28,7 +28,7 @@ interface OwnStateType {
 }
 

```

Comment:
You need to rename the folder to `Loan`.

Reply:
Wait why did you rename some stuff to `loan` but also created another `Loan` folder?

Reply:
I don't understand what you're asking. I'm assuming this has to do with my previous untracked files. If not, could you please clarify?

Reply:
I guess what I don't understand here is why you're changing instances of `Buy` to `Loan` inside the `Buy` tab?

Reply:
I think I know what happened. I renamed some stuff when I was working on loan and it changed files globally. I thought I tracked all of them down and fixed it but some must have gone unchanged.